projects/cloud/environments/environment.prod.ts

```diff
@@ -5,6 +5,10 @@ export const environment = {
     f1EndpointUrl: 'http://54.81.13.248:5408',
     airEndpointUrl:'https://air.labs-air.labs.oocto.eu-west-1.tcie.pro',
     remoteGatewayUrl: '',
-    localGatewayUrl: ''
+    localGatewayUrl: '',
+    issuer_uri: 'https://account.oocto.eu-west-1.tcie.pro',
+    client_id: 'SET_ME_TO_A_VALID_CLIENT_ID',
+    redirect_uri: 'https://air.labs-air.labs.oocto.eu-west-1.tcie.pro/auth/callback',
```

Comment:
Also the route (auth/callback) would be the same in prod and local environment, I think we should be using this in a constant files rather than in an environment file

Reply:
The build is currently copying the environment.prod.ts file during the packaging. You can find this in the angular.json file.